docs/developer-tools/runtime/services/marlowe-tx.md

```diff
@@ -5,22 +5,98 @@ title: Marlowe transaction service
+                  [--contract-host HOST_NAME] [--command-port PORT_NUMBER] 
+                  [-h|--host HOST_NAME] [--submit-confirmation-blocks INTEGER] 
                   [--analysis-timeout SECONDS] [--http-port PORT_NUMBER]
                   --minting-policy-cmd CMD
 
-  Marlowe runtime transaction creation server
+  The transaction creation engine for the Marlowe Runtime. This component exposes
+  a job protocol with four commands: create, apply inputs, withdraw, and submit.
+
+  Create Command
+
+  The create command will create an unsigned transaction body that starts a new Marlowe
+  contract. The options that can be configured for this command are:
+
+    • Adding a staking credential to the Marlowe script address for the contract.
+    • What addresses can be used for coin selection.
+    • Which address to send change to.
+    • Rules for role tokens in the contract. Includes rules for minting and distributing
+      role tokens in the creation transaction and rules for using existing role tokens.
+    • Adding metadata to the transaction
+    • The amount that should be deposited in the contract to cover min UTxO rules
+      (can be omitted and the runtime will compute a worst-case value automatically.
+    • The initial contract or the hash of a contract in the contract store to use as the
+      initial contract.
+
+  Apply Inputs Command
+
+  The apply inputs command will create an unsigned transaction body that advances a
+  Marlowe contract by applying transaction inputs to it. The options that can be
+  configured for this command are:
+
+    • What addresses can be used for coin selection.
+    • Which address to send change to.
+    • Adding metadata to the transaction
+    • The transaction's validity interval. If omitted, the runtime will compute default
+      values from the current contract and the most recent block's slot number.
+    • The initial contract or the hash of a contract in the contract store to use as the
+      initial contract.
+
+  Withdraw Command
+
+  The withdraw command will create an unsigned transaction body that withdraws role payouts
+  from Marlowe contracts from the role payout validators that hold them. When a contract
+  makes a payment to a role, it is not sent directly to the holder of the role token. Instead,
+  it is sent to an auxiliary Plutus script called a payout validator. The holder of the matching
+  role token is then able to spend this script output to withdraw the payout, sending it to an
+  address of their choice. That is what withdrawal means in this context. The options that can be
+  configured for this command are:
+
+    • What addresses can be used for coin selection.
+    • Which address to send change to.
+    • Which payout outputs to withdraw.
+
+  Submit Command
+
+  The submit command will submit a signed transaction to a cardano node via a marlowe-chain-sync
+  instance. It waits until the transaction is confirmed and found in a block by marlowe-chain-sync
```

Comment:
Suggest adding a period at the end of this sentence.

Reply:
```suggestion
  instance. It waits until the transaction is confirmed and found in a block by marlowe-chain-sync.
```